FinSearchUnified/Tests/Helper/UrlBuilderTest.php

```diff
@@ -77,10 +105,50 @@ public function testSendOnlyUniqueUserIps($unfilteredIp, $expectedValue)
     }
 
     /**
+     * @dataProvider reverseProxyIpAddressProvider
+     *
+     * @param string $unfilteredIp
+     *
+     * @throws \Exception
+     */
+    public function testSendsOnlyClientIpFromReverseProxy(string $unfilteredIp)
+    {
+        $this->setIpHeader('HTTP_X_FORWARDED_FOR', $unfilteredIp);
+
+        $urlBuilder = new UrlBuilder($this->httpClient);
+
+        $criteria = new Criteria();
+        $criteria->offset(0)->limit(2);
+
+        $urlBuilder->buildQueryUrlAndGetResponse($criteria);
+        /** @var Zend_Uri_Http $requestedUrl */
+        $requestedUrl = $this->httpClient->getUri()->getQueryAsArray();
+        $usedIpInRequest = $requestedUrl['userip'];
+
+        $this->assertEquals('192.168.0.1', $usedIpInRequest);
+    }
+
+    public function testHandlesUnknownClientIp()
+    {
+        $urlBuilder = new UrlBuilder($this->httpClient);
+
+        $criteria = new Criteria();
+        $criteria->offset(0)->limit(2);
+
+        $urlBuilder->buildQueryUrlAndGetResponse($criteria);
+        /** @var Zend_Uri_Http $requestedUrl */
```

Comment:
This doesn't seem correct anymore. I guess this could be removed since `$requestedUrl` is just an array?